static/sass/6-components/_error-message.scss

```diff
@@ -0,0 +1,33 @@
+.error-message {
```

Comment:
I'd argue this is more a typography style than a component. But I know dealing with type styles is slated for later, so no need to address now.

Reply:
Fair point to bring up. It’s basically just repeating our link styles so it’s not super necessary as a stand-alone anything. I’ll remove it, and apply that look another way. Thanks!

Reply:
![screen shot 2019-01-17 at 10 12 10 am](https://user-images.githubusercontent.com/2974713/51332049-a0d42e80-1a40-11e9-838d-759ef6d7764e.png)

So if I ditch that all together and just slap your fav class `.prose` on the parent, it replicates the desired look, but with a serif. How does that sound in the interim until we have our typography styles refined, @AndrewGibson27?

Reply:
WFM!